src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -125,6 +125,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MetadataUpdaterSupport Condition="'$(MetadataUpdaterSupport)' == '' and '$(Configuration)' != 'Debug'">false</MetadataUpdaterSupport>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Turn off IncludeProjectsNotInAssetsFileInDepsFile by default. -->
+    <IncludeProjectsNotInAssetsFileInDepsFile Condition="'$(IncludeProjectsNotInAssetsFileInDepsFile)' == ''">false</IncludeProjectsNotInAssetsFileInDepsFile>
```

Comment:
I'd suggest changing the default value here to true for the change that goes in 7.0.2xx (the next feature release).

Then we can backport the change to servicing branches, and in those we should set the default to false.

Otherwise, I think this looks all ready to merge.  Thanks!